README.md

```diff
@@ -1 +1,19 @@
-# service-command
\ No newline at end of file
+# EasyEngine/service-command
+
+Manages global `ee-nginx-proxy` container.
+
+## About `ee-nginx-proxy`
+`ee-nginx-proxy` is the main container which routes all incoming request to site-specific containers.
+
+So let's say you have foo.com and bar.com on same ee server. When anyone requests either website, the request will first go to `ee-nginx-proxy` which will forward the request to appropriate nginx container of site (In this case foo.com or bar.com).
+
+## Usage:
+
+```
+ee service start    # starts global container
+ee service restart    # restarts global container
+ee service stop    # stops global container
+ee service reload    # reloads global container
```

Comment:
```
ee service reload             Reload the configuration of the ee-nginx-proxy service
```

Reply:
@mbtamuli I'm thinking about `Reloads the configuration of the global service` since it can contain multiple global containers in future.

Reply:
> multiple global containers in future.

It will have to something like 
```
ee service reload --ee-nginx-proxy
ee service reload --ee-mysql
ee service reload --all
```

Reply:
@mbtamuli Done